src/ios/PurchasesPlugin.swift

```diff
@@ -186,48 +191,42 @@ public class CDVPurchasesPlugin : CDVPlugin {
             return
         }
 
-        RCCommonFunctionality.simulatesAskToBuyInSandbox = askToBuyInSandbox
+        CommonFunctionality.simulatesAskToBuyInSandbox = askToBuyInSandbox
         self.sendOKFor(command: command)
     }
 
     @objc(isAnonymous:)
     func isAnonymous(command: CDVInvokedUrlCommand) {
-        let result = CDVPluginResult(status: .ok, messageAs: RCCommonFunctionality.isAnonymous())
+        let result = CDVPluginResult(status: .ok, messageAs: CommonFunctionality.isAnonymous)
         self.commandDelegate.send(result, callbackId: command.callbackId)
     }
 
-    // NOTE: This method crashes now that it's in Swift. The underlying method call relies on an implicit cast
-    // of a NSDictionary to an RCIntroEligibility object which works in ObjC becasue ObjC doesn't actually care
-    // about types while Swift is type-safe and knows an NSDictionary<String, Any> is not an RCIntroEligibility.
-    // Example crash:
-    // Could not cast value of type '__NSDictionaryI' (0xabacabb) to 'RCIntroEligibility'
-    // This will be fixed once we adopt the newer PurchasesHybridCommon framework (written in Swift, too)
-    // TODO: Remove this warning message once this plugin adopts the newest PurchasesHybridCommon.
     @objc(checkTrialOrIntroductoryPriceEligibility:)
     func checkTrialOrIntroductoryPriceEligibility(command: CDVInvokedUrlCommand) {
         guard let products = command.arguments[0] as? [String] else {
             self.sendBadParameterFor(command: command, parameterNamed: "productIdentifiers", expectedType: NSArray.self)
             return
         }
 
-        RCCommonFunctionality.checkTrialOrIntroductoryPriceEligibility(products) {
+        CommonFunctionality.checkTrialOrIntroductoryPriceEligibility(for: products) {
             let result = CDVPluginResult(status: .ok, messageAs: $0)
             self.commandDelegate.send(result, callbackId: command.callbackId)
         }
     }
 
+    // Update name
     @objc(invalidatePurchaserInfoCache:)
     func invalidatePurchaserInfoCache(command: CDVInvokedUrlCommand) {
-        RCCommonFunctionality.invalidatePurchaserInfoCache()
+        CommonFunctionality.invalidateCustomerInfoCache()
         self.sendOKFor(command: command)
     }
 
     @objc(presentCodeRedemptionSheet:)
     func presentCodeRedemptionSheet(command: CDVInvokedUrlCommand) {
         if #available(iOS 14.0, *) {
-            RCCommonFunctionality.presentCodeRedemptionSheet()
+            CommonFunctionality.presentCodeRedemptionSheet()
         } else {
-            Logger.warn("[Purchases] Warning: tried to present codeRedemptionSheet, but it's only available on iOS 14.0 or greater.")
+            NSLog("%@", "[Purchases] Warning: tried to present codeRedemptionSheet, but it's only available on iOS 14.0 or greater.")
```

Comment:
Is this supposed to be `NSLog`? 🤔

Reply:
> Niiiiccce! I left one comment about a wild `NSLog` that appeared 🤷‍♂️
> 
> Is there also an Android component to adopting hybrid common 3.2.0?

There are a couple jiras for it, but I haven't worked through them yet, focusing on iOS for now while I learn.

Reply:
Re: NSLog- that's how we do logging currently 😄 https://github.com/RevenueCat/purchases-ios/blob/f1044bce14655a7b89423dd33e508963275ad16d/Sources/Logging/Logger.swift#L68